source-code-manager/src/main/java/org/jboss/da/scm/impl/ScmFacade.java

```diff
@@ -38,9 +44,8 @@ public SCMClonner() {
         scmManager.setScmProvider(SCMType.SVN.toString(), new SvnExeScmProvider());
     }
 
-    public void cloneRepository(SCMType scmType, String scmUrl, String revision, File cloneTo)
+    public void shallowCloneRepository(SCMType scmType, String scmUrl, String revision, File cloneTo)
```

Comment:
:+1: Actually I'm even a little bit uncomfortable with the old name. So that sounds like a good compromise